shopinvader_payment/migrations/13.0.1.0.0/post-migrate.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import SUPERUSER_ID, api
+
+
+def _migrate_acquirer(cr):
+    with api.Environment.manage():
+        api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
this is not finished?

Reply:
Was wondering if I needed migrations scripts here. I think not.